src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
+            result = a + b;
+        end
+        
```

Comment:
The add method is straightforward, but consider adding input validation to ensure 'a' and 'b' are numeric.